politeiawww/api/www/v1/api.md

```diff
@@ -2620,6 +2620,49 @@ Reply:
 }
 ```
 
+### `Message Proposer`
```

Comment:
`Message Proposer` doesn't make sense now that this is a user route.  Also, the documentation was not updated.